cmd/harmony/default.go

```diff
@@ -22,6 +22,7 @@ var defaultConfig = harmonyconfig.HarmonyConfig{
 		IsOffline:        false,
 		DataDir:          "./",
 		TraceEnable:      false,
+		MmrDBEnable:      false,
```

Comment:
Do we need a config migration for this? When would (external) validators want to use this flag?

Reply:
may not. most nodes do not need to use this flag, only 1 or 2 nodes need to enable this flag to provide MMR proof queries.